Huawei/PIC-LX9/Android.mk

```diff
@@ -0,0 +1,8 @@
+LOCAL_PATH := $(call my-dir)
+include $(CLEAR_VARS)
+LOCAL_MODULE_TAGS := optional
+LOCAL_PACKAGE_NAME := treble-overlay-huawei-PIC
```

Comment:
Please give it (and folder) the commercial name.
I guess it's Huawei Nova 2 according to google?

Reply:
yes, it's nova2